### ISSUE TITLE
As a user setting up a Gitcoin Grant, I'm stuck in no man's land - on the second try my grant went through, but I have no subscribers and no contributions in the Grants UI, and I can't cancel to redo the subscription.

### ISSUE THREAD

**Issue by frankchen07** [gitcoinco/web#3415]
### User Story

As a user setting up a Gitcoin Grant, I'm stuck in no man's land - on the second try my grant went through, but I have no subscribers and no contributions in the Grants UI, and I can't cancel to redo the subscription.

### Why Is this Needed

[comment]: # (Describe the problem and why this task is needed. Provide description of the current state, what you would like to happen, and what actually happen)
*Summary*:

### Description

Under Appendix 1: https://docs.google.com/document/d/1acaxboTHVqo4rM7BJ5e2laZpxe45qJs3oCNnp9tAlEo/edit#

TLDR (but you should read it)

1. User had to try twice to submit the grant contribution to Gitcoin. Both succeeded.
2. `executeSubscription` on the transaction has failed, a couple hundred blocks after both initial `approve` transactions.
3. Gitcoin Grants UI says the user is a subscriber, but the Grants screen is showing no subscribers and no contributions.
4. User attempts to cancel, cancellation goes through ok.
5. However, the user is in no-man's land. Can't cancel, can't redo the contribution without cancelling. The allowance of DAI is also 0, so the DAI can't leave his account.

### Expected Behavior

TBD 

### Definition of Done

TBD

**Comment by captnseagraves** [comment]
Just created this issue https://github.com/gitcoinco/web/issues/3419 which I believe will address the root of the users problem in this case.
